### ISSUE TITLE
Autoinstall - a very simple and important feature

### ISSUE THREAD

**Issue by ttodua** [xdebug/vscode-php-debug#817]
Hi, thanks for extension.
However, there is a very simple and important thing you can do - in readme you describe the whole process of installing xdebug manually.  it's 2022 and we still are doing the installations and etc manually, so, if you can, please take a step forward and do this if you can:

1) check php `PATH`  environment variable to determine the `php` path
2) run `phpinfo()` in the background
3) send output to `https://xdebug.org/wizard`
4) download the suggested file and move to the needed location, rename file
5) update php.ini as written there.

That will do the job. which 7million ppl seems to be doing manually.
thanks.

**Comment by zobo** [comment]
Hi @ttodua, thanks for the message. 

Indeed this is on the "roadmap", almost exactly as you described it. I already started talks with Derick of Xdebug to implement an API interface for checks against the wizard page. 

On windows this is a matter of downloading a .dll. On Linux systems it's usually a matter of the right package manager. On MacOS maybe brew?

This is the related task https://github.com/xdebug/vscode-php-debug/issues/672

**Comment by ttodua** [comment]
Exactly, on first thought I wondered whether there might appear api-like endpoint for it, and good you have talked with him. hoping that day will come soon. even though that  'wizard' page kinda made process a bit easier for ppl, still, enabling the debugger and downloading packages manually and the setup process... has been struggling for decades.

about OS - I am mostly on Windows and can't say about other packages to be honest, but I hope you will find a correct path for that.
Thanks for the work.
